docs/general/fields.md

```diff
@@ -0,0 +1,23 @@
+[Back to Guides](../README.md)
+
+# Fields
+
+If for a reason, you need to override the fields defined on serializer, you should use `fields`
```

Comment:
```diff
-If for a reason, you need to override the fields defined on serializer, you should use `fields`
+If, for any reason, you would like to return only specific fields, you should use the `fields` option.
```

Here we're not actually override the fields (e.g. changing their value) but restricting them.